sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Extensions/WebPubSubEndpointRouteBuilderExtensions.cs

```diff
@@ -16,7 +16,8 @@ public static class WebPubSubEndpointRouteBuilderExtensions
         /// <summary>
         /// Maps the <see cref="WebPubSubHub"/> to the path <paramref name="path"/>.
         /// </summary>
-        /// <typeparam name="THub">User implemented <see cref="WebPubSubHub"/>.</typeparam>
+        /// <typeparam name="THub">User implemented <see cref="WebPubSubHub"/>.
+        /// Name of the class has to match the name of the hub in the Azure portal.</typeparam>
         /// <param name="endpoints">The <see cref="IEndpointRouteBuilder"/>.</param>
         /// <param name="path">The path to map the <see cref="WebPubSubHub"/>.</param>
         /// <returns>The <see cref="IEndpointConventionBuilder"/>.</returns>
```

Comment:
this method can be simplified to call below one.

```
public static IEndpointConventionBuilder MapWebPubSubHub<THub>(
            this IEndpointRouteBuilder endpoints,
            string path) where THub: WebPubSubHub
{
    return MapWebPubSubHub<THub>(endpoints, path, nameof(THub));
}
```

Reply:
sure, will do!

Reply:
I think the sample you provided is slightly wrong - it's not `nameof(THub)` which is `"THub"` but `typeof(THub).Name`

Reply:
I've cleaned up the code. Thanks for the comment - you were right that it removes a lot of duplicate code